src/cirrus/release.py

```diff
@@ -329,12 +344,45 @@ def _trigger_jenkins_release(config, new_version, level):
         raise RuntimeError('Jenkins HTTP API returned code {}'.format(response.status_code))
 
 
+def wait_on_gh_status(branch_name, timeout=600):
+    """
+    _wait_on_gh_status_
+
+    Wait for CI checks to complete for the branch named
+    """
+    time_spent = 0
+    status = branch_status(branch_name)
+    while status == 'pending':
+        if time_spent > timeout:
+            LOGGER.error("Exceeded timeout for branch status {}".format(branch_name))
+            break
+        status = branch_status(branch_name)
+        time.sleep(2)
+        time_spent += 2
```

Comment:
Can we make this configurable as well? For projects where the CI check is normally on the order of minutes 2 seconds seems overly frequent